packages/flutter/lib/src/painting/borders.dart

```diff
@@ -294,6 +313,25 @@ class BorderSide {
     );
   }
 
+  /// Get the amount of the stroke width that lies inside of the [BorderSide].
+  ///
+  /// For example, this will return the [wdith] for a [strokeAlign] of -1, half
+  /// the [width] for a [strokeAlign] of 0, and 0 for a [strokeAlign] of 1.
+  double get strokeInset => width * (1 - (1 + strokeAlign) / 2);
+
+  /// Get the amount of the stroke width that lies outside of the [BorderSide].
+  ///
+  /// This will return 0 for a [strokeAlign] of -1, half the [width] for a
```

Comment:
You might add the "For example, " at the beginning of these last two, just so they know that those aren't the only three values it returns.

Reply:
Done